CONTRIBUTING.md

```diff
@@ -350,6 +350,64 @@ Notice how we do all of these things manually:
 
 We have full access to the [Cypress API](https://docs.cypress.io/api) so any additional interactions or custom logic can be easily added.
 
+### Accessibility testing
+
+We reuse our component examples for accessibility testing with
+a Cypress script that evaluates each of their iframes for accessibility
+violations. 
+
+#### Running accessibility tests
+
+In the terminal: 
+
+- Run the command `yarn workspace a11y test` to run all of the component tests in the `a11y\src` folder. This will include `Component.cy.tsx`, which tests all of the component examples in the `iTwinUI\examples` directory.
+
+In the Cypress GUI:
+
+1. From the monorepo root, run `yarn workspace a11y open`. This will open the Cypress control panel where you can run the tests.
+2. Choose a browser to evaluate your tests through, then press the `Start Component Testing in [YourBrowser]` button below.
+3. Select `Component.cy.tsx` to run the script that tests all of the component examples. Any additional component test you write and place into `a11y\src` will also be present here for testing.
+
+##### Your results
+
+In the terminal, a table will be produced for each violating component with the Axe rule ID being violated and its description. 
+
+In Cypress, if the component violates a Axe rule, its rule ID and the number of offending nodes will be output in a line in the testing window. You can click on the line to highlight the offending nodes in the test browser. 
```

Comment:
Mention that clicking on a line will also output more info in the browser console.